src/editor/Core/Utility.re

```diff
@@ -89,3 +89,17 @@ let escapeSpaces = str => {
   let whitespace = Str.regexp(" ");
   Str.global_replace(whitespace, "\\ ", str);
 };
+
+let filterMap = (f, l) => {
```

Comment:
I actually found out recently that there is a `List.filter_map` in the standard lib